src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx

```diff
@@ -0,0 +1,111 @@
+import { useState } from 'react';
+import { validateAddress } from '@polkadot/util-crypto';
+
+import { useI18n } from '@app/providers';
+import { Input, Combobox, Identicon, Icon, IconButton } from '@shared/ui';
+import { signatoryModel } from '../../../model/signatory-model';
+
+interface Props {
+  index: number;
+  canBeDeleted?: boolean;
+  onDelete?: (index: number) => void;
+}
+
+export const Signatory = ({ index, onDelete, canBeDeleted = true }: Props) => {
+  const { t } = useI18n();
+  const [name, setName] = useState('');
+  const [address, setAddress] = useState('');
+  const [query, setQuery] = useState('');
+
+  const onNameChange = (newName: string) => {
+    setName(newName);
+    signatoryModel.events.signatoriesChanged({
+      index,
+      name: newName,
+      address,
+    });
+  };
+
+  const onAddressChange = (newAddress: string) => {
+    try {
+      validateAddress(newAddress);
+      setAddress(newAddress);
+      signatoryModel.events.signatoriesChanged({
+        index,
+        name,
+        address: newAddress,
+      });
+    } catch {
+      // do nothing wrong address
+      setAddress('');
+    }
+  };
+
+  // const options = proxyAccounts.map((proxyAccount) => {
```

Comment:
add todo when it should be uncommented or remove it

Reply:
Absolutely, sorry about that, there are still a lot of comments, because for instance I don't know yet how to cater for proxy accounts, or shard accounts etc. I'll remove them.